config/initializers/simple_form.rb

```diff
@@ -0,0 +1,169 @@
+# Use this setup block to configure all options available in SimpleForm.
+SimpleForm.setup do |config|
```

Comment:
This is an initializer file for the `SimpleForm` library I'm using to help build our forms. Don't be too concerned about it at this point.

Reply:
so I've read about SimpleForm, but it seems that people have mixed feelings about it- what would you say are its advantages? 

Is it okay to go with RoR native form builder?

Reply:
It's perfectly find to start with the Rails native form builder and helpers, that's fine. Simple form mostly follows the same API, but it makes it real quick and easy to build forms. In terms of benefits, you configure it however you want and it automatically generates the html around each input field based on your settings. Also, it automatically handles error messages and makes things really easy to configure. We use it for our forms at work and have seen great success/flexibility.